utils/wasm-instrument/src/tests.rs

```diff
@@ -183,7 +191,8 @@ fn duplicate_export() {
     );
     let module = parse_wat(&wat);
 
-    assert!(inject(module, &ConstantCostRules::default(), "env").is_err());
+    let (gr_system_break_index, module) = inject_system_break_import(module, "env").unwrap();
```

Comment:
don't you want to make them some kind of builder so nobody will missuse the thing that they always supposed to be inserted together one by one?

Reply:
I can try to do it in this PR, but it looks like it will much  change the gear-wasm-instrument

Reply:
done in last commits